pkg/metrics/metrics.go

```diff
@@ -210,6 +210,14 @@ var (
 	// Policy is the number of policies loaded into the agent
 	Policy = NoOpGauge
 
+	// PolicyUsed
+	// PolicyUsed is the number of used policies loaded into the agent
+	PolicyUsed = NoOpGauge
+
+	// PolicyUnused
+	// PolicyUnused is the number of unused policies loaded into the agent
+	PolicyUnused = NoOpGauge
+
```

Comment:
It seems to me that `PolicyUsed` + `PolicyUnused` should always be equal to `Policy`. Do we have confidence that this is the case? Also, if this is the case, do we really need both of them as `PolicyUnused` could always be derived from `Policy` - `PolicyUsed`?